src/edu/washington/escience/myria/operator/TupleRangeSource.java

```diff
@@ -1,14 +1,15 @@
 package edu.washington.escience.myria.operator;
```

Comment:
I think we can just remove `TupleRangeSource` since it's only used in `TestCounter` but there is no need to use such a tuple source. The test only tests on the second column ("Counter"). It can be replaced by any tuple source in other tests.